.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: php
 
+env:
+  global:
+    - CACHE_DIR=$HOME/.mink-debug-cache
```

Comment:
Okay, let's stick to `MINK_DEBUG_CACHE_DIR`, it's quite long, but `SYLIUS_CACHE_DIR` makes sense in Sylius.